### PR TITLE
Falcon unzip

### PR DESCRIPTION
**Pull request by xfzhao**

### CODE REVIEW

File: var/spack/repos/builtin/packages/falcon-unzip/package.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    homepage = "https://github.com/PacificBiosciences/FALCON_unzip/wiki"
+    version('2018.31.08-03.06-py2.7-ucs4-beta',
+            sha256='bd6a29cc25f39924aad0137da3f1be63dce60f16b0bd044d58728b8407460545',
+            url='https://downloads.pacbcloud.com/public/falcon/falcon-2018.03.12-04.00-py2.7-ucs4.tar.gz')
+
+    depends_on('python@2.7.0:2.7.99',  type='run')
+    depends_on('mummer',      type='run')
+    depends_on('samtools',    type='run')
+    depends_on('minimap2',    type='run')
+
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/falcon-unzip/package.py]: do we need pbalign and variantcaller that are mentioned in the wiki ?

File: var/spack/repos/builtin/packages/py-pauvr/package.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPauvr(PythonPackage):
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/py-pauvr/package.py]: this seems to have snuck in, and needs removed from this branch

File: var/spack/repos/builtin/packages/falcon-unzip/package.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+class FalconUnzip(Package):
+    """FALCON-Unzip works with FALCON  for full diploid assembly."""
+
+    homepage = "https://github.com/PacificBiosciences/FALCON_unzip/wiki"
+    version('2018.31.08-03.06-py2.7-ucs4-beta',
+            sha256='bd6a29cc25f39924aad0137da3f1be63dce60f16b0bd044d58728b8407460545',
+            url='https://downloads.pacbcloud.com/public/falcon/falcon-2018.03.12-04.00-py2.7-ucs4.tar.gz')
+
+    depends_on('python@2.7.0:2.7.99',  type='run')
```
Comment (baberlevi) [inline var/spack/repos/builtin/packages/falcon-unzip/package.py]: based on https://github.com/PacificBiosciences/FALCON_unzip/wiki/Binaries#which-binary-do-i-need
I think this dependency needs to be on python+ucs4

File: var/spack/repos/builtin/packages/falcon-unzip/package.py (hunk 1 above)
Comment (xfzhao) [reply var/spack/repos/builtin/packages/falcon-unzip/package.py]: 1- for making draft , need FALCON based on the website
2- to refine , need pbalign and variantcaller, samtools. will create new packages for  pbalign and variantcaller.
3. needs py2.7+ucs4

FALCON-unzip will be updated next week after packing pbalign and variantcaller.

Comment (scheibelp) [reply var/spack/repos/builtin/packages/falcon-unzip/package.py]: @xfzhao are you going to make these changes in this PR? Or do you think the PR is ready?

Comment (alalazo) [reply var/spack/repos/builtin/packages/falcon-unzip/package.py]: @xfzhao :point_up:

**Comment by alalazo** [comment]
Closing as stale

**Pull request closed without merging**